app/src/main/java/tranquvis/simplesmsremote/CommandManagement/Modules/ModuleAuthentication.java

```diff
@@ -0,0 +1,23 @@
+package tranquvis.simplesmsremote.CommandManagement.Modules;
+
+import android.Manifest;
+import android.os.Build;
+
+import tranquvis.simplesmsremote.Activities.ModuleActivities.AuthenticationModuleActivity;
+import tranquvis.simplesmsremote.Activities.ModuleActivities.CameraModuleActivity;
+import tranquvis.simplesmsremote.CommandManagement.Commands.CommandAuth;
+import tranquvis.simplesmsremote.CommandManagement.Commands.CommandGetAudioVolume;
+import tranquvis.simplesmsremote.CommandManagement.Commands.CommandSetAudioVolume;
+import tranquvis.simplesmsremote.R;
+
+public class ModuleAuthentication extends Module {
+    public final CommandAuth commandAuth = new CommandAuth(this);
+
+    public ModuleAuthentication() {
+        this.titleRes = R.string.control_module_title_authentication;
```

Comment:
Ya, true. I used "Grant Phone Remotely" as the module name. Bit long, but I did want to show that it's remote management and not just managing granted phones on your phone somehow (although that wouldn't make sense as an sms module, but still). Also didn't include the "manage" part because you can - currently - only grant new phones, not manage old ones.